moz_kinto_publisher/test_publisher.py

```diff
@@ -87,108 +94,113 @@ def test_load_remote(self):
 
 
 class TestPublishDecisions(unittest.TestCase):
+    # Create a new temporary directory and cd into it for each test.
```

Comment:
So are the temporary directories cleaned up automatically?

Reply:
Yes, everything here ends up living in pytest's "[base temporary directory](https://docs.pytest.org/en/6.2.x/tmpdir.html#the-default-base-temporary-directory)".